code/game/machinery/rechargestation.dm

```diff
@@ -154,6 +154,18 @@
 		else if(default_part_replacement(user, O))
 			return
 
+	if(istype(O, /obj/item/grab))
+		var/obj/item/grab/grab = O
+		var/mob/living/L = grab.affecting
+		if(!L.isSynthetic())
+			return
+		
+		user.visible_message(SPAN_NOTICE("[user] starts putting [L] into [src]."), SPAN_NOTICE("You start putting [L] into [src]."), range = 3)
```

Comment:
The code below here that you added does the same thing as some of the code you added to `/obj/machinery/recharge_station/MouseDrop_T` -- But you have better error messages in `MouseDrop_T`. You should extract all the common code into another proc and call it in both locations.

Reply:
Done as well. It should be better now